### DISCUSSION TITLE
How to I delete / unfollow one of my own playlists

### DISCUSSION THREAD

**Discussion by KenthJ** [JohnnyCrazy/SpotifyAPI-NET#660] [Q&A] [answered]
How to I delete / unfollow one of my own playlists, can't seem to find any method for this, and google was not much of a help

I found a four year old thread where another user asked for how to delete a playlist, and the response here was that it was not possible, and you should unfollow that specific playlist instead, but I also can't find any method for unfollowing some of my own playlist

Thanks a lot in advance

**Comment by JohnnyCrazy** [comment, marked as answer]
https://github.com/JohnnyCrazy/SpotifyAPI-NET/blob/master/SpotifyAPI.Web/Clients/Interfaces/IFollowClient.cs

```c#
var spotify = new SpotifyClient("AccessToken");

spotify.Follow.UnfollowPlaylist("YourPlaylistId")
```

**Reply by KenthJ** [reply]
Thanks a lot, worked like a charm
